regulations/views/comment.py

```diff
@@ -70,3 +71,29 @@ def submit_comment(request):
         'status': 'submitted',
         'metadata_url': metadata_url,
     })
+
+
+@csrf_exempt
+@require_http_methods(['GET', 'HEAD'])
+def get_federal_agencies(request):
+    url = "{}?{}".format(settings.REGS_GOV_API_LOOKUP_URL,
+                         "field=gov_agency&dependentOnValue=Federal")
```

Comment:
Prefer not to URL-encode query strings manually--this looks fine, but IMO it's error-prone in general. Requests will do this for us:

```python
requests.get(url, params={'field': 'gov_agency', ...})
```